### ISSUE TITLE
Better arrangement choices for a frontend to enumerate pick-list options

### ISSUE THREAD

**Issue by theflyingape** [libretro/RetroArch#17004]
In reference to a _post by @sonninnos in https://github.com/libretro/RetroArch/pull/16960#discussion_r1759653679_

Options that might get packaged with RetroArch, like a theme color, font, default core overlay, global shader, menu background or navigation sounds, screensaver, etc.

I've always liked those GUI pick-list items for community-based or plug-in options that sort by a "popularity" rating, then by date "added" -- allows for an author to put the first 5-star vote on a release so it sorts at the top until subsequent votes come in, instead of it falling unceremoniously to the bottom when it is appended.

Internally, the option continues as a fixed integer for backward compatibility, but obviously, the change is adding the metadata (and a community popularity poll database) for frontend(s) to implement this arrangement -- also motivates frontend(s) to modernize. If nothing else changes, it simply continues to work AS-IS.

**Comment by RobLoach** [comment]
What sonninnos referenced is switching the integer in the config to a string for the ozone color theme. Are you looking for an overview of which themes are available?

**Comment by theflyingape** [comment]
Exactly. Keep the configuration integers intact (don't break anything), but extend those pick-list items beyond a description label by introducing some supporting metadata that _allows_ for a frontend to present them differently, such as ordering.
